configurations/test/env.yml

```diff
@@ -1,6 +1,7 @@
 AUTH0_API_CLIENT: test-auth0-client-id
 AUTH0_API_SECRET: test-auth0-secret
 AUTH0_DOMAIN: test.auth0.com
+DISABLE_AUTH: true
```

Comment:
Insert a line for the `AWS_PROFILE` parameter, or mention that in the PR description. That parameter is required to make the new tests pass.

Reply:
@binh-dam-ibigroup , yeah thanks I missed this. I have updated the PR description.